### PR TITLE
Created a 404 for fans of John Magikarp

### PR DESCRIPTION
**Pull request by DrTexxOfficial**
I got ya covered Terence

### CODE REVIEW

**Comment by terencehuynh** [comment]
Solves #52

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+	<title>Junior Dev - :c</title>
+	<meta charset="utf-8">
+    <meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0' name='viewport' />
+    <link rel="stylesheet" href="./styles.css" media="screen" title="no title">
+    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha256-/SIrNqv8h6QGKDuNoLGA4iret+kyesCkHGzVUUV0shc=" crossorigin="anonymous"></script>
```
Comment (terencehuynh) [inline 404.html]: Probably remove this - we're not using jQuery in this page. Save yourself an unnecessary download :)

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+</head>
+<body>
+<div class="container">
+	<div class="container-content">
+		<div id="mainBody" style="height: 100vh; overflow: hidden;">
+			<div>
+				<h2>[404] Looks like you've swum a little too far :(</h2>
+				<p>Unfortunately we can't find the page you're looking for... <a href="index.html">swim back?</a></p>
+			</div>
+			<img style="height: 100%; padding: 0; margin: auto;" src="https://johnjohns1.fjcdn.com/comments/Pic+very+related+to+this+post+_4168c7fde92a08c0f8519fcb4506303e.jpg">
```
Comment (terencehuynh) [inline 404.html]: Let's see this image in all of its glory. Change this to:

```suggestion
			<img style="width: 100%; height: auto; padding: 0; margin: auto;" src="https://johnjohns1.fjcdn.com/comments/Pic+very+related+to+this+post+_4168c7fde92a08c0f8519fcb4506303e.jpg">
```

Just so people can see the whole image 👍

**Comment by nm-t** [comment]
Thanks @DrTexxOfficial and @terencehuynh! Looks.. troubling..

**Pull request merged**